flux-common/src/config.rs

```diff
@@ -54,6 +55,7 @@ pub static CONFIG: LazyLock<Config> = LazyLock::new(|| {
             .set_default("check_asserts", "assume")?
             .set_default("check_asserts", "assume")?
             .set_default("pointer_width", 64)?
+            .set_default("check_target", "")?
```

Comment:
I don't like the word target because it can be confused with the target architecture/os (that's what I thought when first reading the title). Maybe something with `filter` or `def`, `LR_CHECK_DEF`, `LR_DEF_FILTER`

Reply:
Yes `LR_CHECK_DEF` is good!